pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test: TestTenantLogic_create_as failed

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_create_as [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5915660?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5915660?buildTab=artifacts#/) on master @ [cd113bf428e8868c38b2f2f295210f5c30d13a2d](https://github.com/cockroachdb/cockroach/commits/cd113bf428e8868c38b2f2f295210f5c30d13a2d):


```
=== RUN   TestTenantLogic_create_as
    test_log_scope.go:162: test logs captured to: /artifacts/tmp/_tmp/9eebf1fbe77b8b038ea96f01f08fb077/logTestTenantLogic_create_as2700870442
    test_log_scope.go:80: use -show-logs to present logs inline
[05:26:01] rng seed: -3397463098033268904

[05:26:03] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3135/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/create_as: 50 statements
[05:26:25] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3135/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/create_as: 78 statements
[05:26:25] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3135/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/create_as with config 3node-tenant: 79 tests, 0 failures
    logic.go:3574: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3135/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/create_as:397: error while processing
    logic.go:3574: test was successful but validation upon completion failed: dropping database defaultdb failed: pq: polling for queued jobs to complete: poll-show-jobs: liveness session expired 1.344599008s before transaction
    panic.go:500: -- test log scope end --
--- FAIL: TestTenantLogic_create_as (91.95s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_create_as.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-18199

--- a/pkg/ccl/logictestccl/tests/3node-tenant/generated_test.go
+++ b/pkg/ccl/logictestccl/tests/3node-tenant/generated_test.go
@@ -463,6 +463,13 @@ func TestTenantLogic_create_as(
 	runLogicTest(t, "create_as")
 }
 
+func TestTenantLogic_create_as_non_metamorphic(
+	t *testing.T,
+) {
+	defer leaktest.AfterTest(t)()
+	runLogicTest(t, "create_as_non_metamorphic")
+}
+
 func TestTenantLogic_create_index(
 	t *testing.T,
 ) {

--- a/pkg/sql/logictest/testdata/logic_test/create_as
+++ b/pkg/sql/logictest/testdata/logic_test/create_as
@@ -382,17 +382,3 @@ query I
 SELECT * FROM tab_from_seq
 ----
 2
-
-# Regression test for #81554, where tried to do gigantic batches for CTAS in
-# explicit transactions. Use a fixed command size, so that an error is decoupled
-# fom the default size.
-statement ok
-SET CLUSTER SETTING kv.raft.command.max_size='5m'
-
-statement ok
-BEGIN;
-CREATE TABLE source_tbl_huge AS SELECT 1::CHAR(256) FROM generate_series(1, 500000);
-COMMIT;
-
-statement ok
-SET CLUSTER SETTING kv.raft.command.max_size to default

--- a/pkg/sql/logictest/testdata/logic_test/create_as_non_metamorphic
+++ b/pkg/sql/logictest/testdata/logic_test/create_as_non_metamorphic
@@ -1,0 +1,16 @@
+# LogicTest: !metamorphic
+# Disabled to allow us to validate create as with large batch sizes.
+
+# Regression test for #81554, where tried to do gigantic batches for CTAS in
+# explicit transactions. Use a fixed command size, so that an error is decoupled
+# fom the default size.
+statement ok
+SET CLUSTER SETTING kv.raft.command.max_size='5m'
+
+statement ok
+BEGIN;
+CREATE TABLE source_tbl_huge AS SELECT 1::CHAR(256) FROM generate_series(1, 500000);
+COMMIT;
+
+statement ok
+SET CLUSTER SETTING kv.raft.command.max_size to default

--- a/pkg/sql/logictest/tests/fakedist-disk/generated_test.go
+++ b/pkg/sql/logictest/tests/fakedist-disk/generated_test.go
@@ -436,6 +436,13 @@ func TestLogic_create_as(
 	runLogicTest(t, "create_as")
 }
 
+func TestLogic_create_as_non_metamorphic(
+	t *testing.T,
+) {
+	defer leaktest.AfterTest(t)()
+	runLogicTest(t, "create_as_non_metamorphic")
+}
+
 func TestLogic_create_index(
 	t *testing.T,
 ) {

--- a/pkg/sql/logictest/tests/fakedist-vec-off/generated_test.go
+++ b/pkg/sql/logictest/tests/fakedist-vec-off/generated_test.go
@@ -436,6 +436,13 @@ func TestLogic_create_as(
 	runLogicTest(t, "create_as")
 }
 
+func TestLogic_create_as_non_metamorphic(
+	t *testing.T,
+) {
+	defer leaktest.AfterTest(t)()
+	runLogicTest(t, "create_as_non_metamorphic")
+}
+
 func TestLogic_create_index(
 	t *testing.T,
 ) {

--- a/pkg/sql/logictest/tests/fakedist/generated_test.go
+++ b/pkg/sql/logictest/tests/fakedist/generated_test.go
@@ -436,6 +436,13 @@ func TestLogic_create_as(
 	runLogicTest(t, "create_as")
 }
 
+func TestLogic_create_as_non_metamorphic(
+	t *testing.T,
+) {
+	defer leaktest.AfterTest(t)()
+	runLogicTest(t, "create_as_non_metamorphic")
+}
+
 func TestLogic_create_index(
 	t *testing.T,
 ) {

--- a/pkg/sql/logictest/tests/local-vec-off/generated_test.go
+++ b/pkg/sql/logictest/tests/local-vec-off/generated_test.go
@@ -436,6 +436,13 @@ func TestLogic_create_as(
 	runLogicTest(t, "create_as")
 }
 
+func TestLogic_create_as_non_metamorphic(
+	t *testing.T,
+) {
+	defer leaktest.AfterTest(t)()
+	runLogicTest(t, "create_as_non_metamorphic")
+}
+
 func TestLogic_create_index(
 	t *testing.T,
 ) {

--- a/pkg/sql/logictest/tests/local/generated_test.go
+++ b/pkg/sql/logictest/tests/local/generated_test.go
@@ -450,6 +450,13 @@ func TestLogic_create_as(
 	runLogicTest(t, "create_as")
 }
 
+func TestLogic_create_as_non_metamorphic(
+	t *testing.T,
+) {
+	defer leaktest.AfterTest(t)()
+	runLogicTest(t, "create_as_non_metamorphic")
+}
+
 func TestLogic_create_index(
 	t *testing.T,
 ) {

